startup/gui/renderPassEditor.py

```diff
@@ -113,3 +113,13 @@
 	GafferSceneUI.RenderPassEditor.registerOption( "Arnold", "ai:GI_transmission_depth", "Ray Depth", "Transmission" )
 	GafferSceneUI.RenderPassEditor.registerOption( "Arnold", "ai:GI_volume_depth", "Ray Depth", "Volume" )
 	GafferSceneUI.RenderPassEditor.registerOption( "Arnold", "ai:auto_transparency_depth", "Ray Depth", "Transparency" )
+
+# Register the default grouping function used to display render passes in a hierarchy.
+# This groups render passes based on the first token in their name delimited by "_".
+def __defaultPathGroupingFunction( renderPassName ) :
+
+	return IECore.InternedStringVectorData(
```

Comment:
Sorry for the separate comment outside the original review. I'm wondering if we should drop the requirement to return InternedStringVectorData here in favour of `/string/with/slashes' and/or `[ "list", "of", "python", "strings" ]`. InternedStringVectorData isn't especially Pythonic. Actually, I think the former would happen magically if you `extract<vector<InternedString>>()` because there's a string->ScenePath converter already. And maybe we should just register a list->ScenePath converter too.

What made me think of this is that @ericmehl has a similar presentation conundrum in his "path remapping based on kind" thing in the SelectionTool. Would be good to make sure they line up...

Reply:
> Sorry for the separate comment outside the original review...

No problem! After our offline discussion I've gone ahead and dropped the InternedStringVectorData requirement. It does feel like the right direction to head in general. 

I went ahead and registered the list->ScenePath converter in 2683507 (new territory, but it appears to do the trick) and reordered the commits so it comes first, everything else is unchanged apart from the fixups. Even with that in place I found that I also favoured the paths as strings approach. From a user perspective `/string/with/slashes` seems more pathy than `[ "list", "of", "python", "strings" ]`, and we have `GafferScene.ScenePlug.stringToPath` to help there. 288eaa3, 015d3ef and 05298e8 have the remaining changes - I left one test using paths as strings and the other as paths as a list of strings to exercise both options.

> The grouping/ungrouping process can be a little bit disorienting, and maintaining selection would give you a sort of visual anchor to help reorient by.

Agreed, seeing as we've come up with a pretty clear path for inclusions/exclusions I've snuck in the selection remapping in b00eb2f. All that's missing now is `F` to expand and frame like we have in the HierarchyView...